ontology/spdx-ontology.owl.xml

```diff
@@ -1455,13 +1351,6 @@ Known issues:</rdfs:comment>
                 <owl:onDataRange rdf:resource="http://www.w3.org/2001/XMLSchema#string"/>
             </owl:Restriction>
         </rdfs:subClassOf>
-        <rdfs:subClassOf>
-            <owl:Restriction>
-                <owl:onProperty rdf:resource="http://www.w3.org/2000/01/rdf-schema#comment"/>
-                <owl:maxQualifiedCardinality rdf:datatype="http://www.w3.org/2001/XMLSchema#nonNegativeInteger">1</owl:maxQualifiedCardinality>
-                <owl:onDataRange rdf:resource="http://www.w3.org/2001/XMLSchema#string"/>
-            </owl:Restriction>
-        </rdfs:subClassOf>
```

Comment:
Can you say more about how the code generator works.

Reply:
There are 3 projects that use the RDF OWL document for checking restrictions:
- [spdx-java-rdf-store](https://github.com/spdx/spdx-java-rdf-store) supports the serialization and deserialization of SPDX files in RDF format
- [OwlToJsonSchema](https://github.com/spdx/tools-java/blob/master/src/main/java/org/spdx/tools/schema/OwlToJsonSchema.java) converts the OWL ontology into a JSON schema
- [OwlToJsonContext](https://github.com/spdx/tools-java/blob/master/src/main/java/org/spdx/tools/schema/OwlToJsonContext.java) converts the OWL ontology into a JSON schema format

Reply:
These projects use the restrictions in the OWL document to associate properties with classes and to determine the cardinality restrictions on those properties.

Reply:
In looking at the restrictions - since they are nested inside `<owl:Class rdf:about="http://spdx.org/rdf/terms#CreationInfo">`, would that limit the scope of this restriction to the `rdf-schema#comment` within this class or does this restriction end up being applied to all `rdf-schema#comment` uses?